client/lib/abtest/active-tests.js

```diff
@@ -130,13 +130,15 @@ export default {
 		},
 		defaultVariation: 'original',
 	},
-	onlyJetpackMonthly: {
-		datestamp: '20190212',
+	jetpackMontlyPlansOnly: {
```

Comment:
Ugh, we have a typo here: `jetpackMontlyPlansOnly` should be `jetpackMonthlyPlansOnly`

Reply:
shit, I copy/pasted the typo everywhere :man_facepalming:

Reply:
Well, at least you can fix it in this PR, including the update to the known e2e tests :)